### PR TITLE
Changed String.format to String template

### PR DESCRIPTION
**Pull request by SanjaySargam**
## Pull Request template
Changed String.format to String template

## Fixes
Bug #11449 

## Checklist
_Please, go through these checks before submitting the PR._

- [ ] You have a descriptive commit message with a short title (first line, max 50 chars).
- [ ] You have commented your code, particularly in hard-to-understand areas
- [ ] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/test/java/com/ichi2/anki/TestUtils.kt [hunk 1]
```diff
@@ -31,7 +34,7 @@ open class TestUtils {
             md.update(getFileContentsBytes(File(filename)))
             val hex = StringBuilder()
             for (b in md.digest()) {
-                hex.append(String.format("%02x", b))
+                hex.append(b.format())
```
Comment (lukstbit) [inline AnkiDroid/src/test/java/com/ichi2/anki/TestUtils.kt:37]: Please revert this change. We shouldn't create a custom function to maintain if there's a platform function(System.format) already available.

**Review by lukstbit — CHANGES_REQUESTED** [review]

File: AnkiDroid/src/test/java/com/ichi2/anki/TestUtils.kt (hunk 1 above)
Comment (SanjaySargam) [reply AnkiDroid/src/test/java/com/ichi2/anki/TestUtils.kt:37]: ok Fine

**Review by lukstbit — APPROVED** [review]
